schema_utils.py

```diff
@@ -454,6 +473,7 @@ def is_at_least(obj, min_value):
 
     @staticmethod
     def is_at_most(obj, max_value):
+        # type: (float, float) -> bool
```

Comment:
No that was just an example to show similarity between Unicode and str. I kept float here because why can't we specify the max-value of float to be float.

Reply:
I don't see any problem in that, but let's keep the mentioned type only if there aren't other reasons.

Reply:
@hardikkat24 any update on this?

Reply:
@hardikkat24 Any update on this?

Reply:
@sajalasati Sorry, I forgot about this. Thanks!